### ISSUE TITLE
Documentation page is broken

### ISSUE THREAD

**Issue by polomarcus** [siddii/angular-timer#157]
http://siddii.github.io/angular-timer/ cannot load this file http://siddii.github.io/angular-timer/app/js/_timer.js

**Comment by siddii** [comment]
Yes, there are some issues with file paths after the PR merege. I reverted back to the previous commit. 

Need to spend time later cleaning it up.
